infrastructure/helm/quantumserverless/charts/gateway/templates/job.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: batch/v1
+kind: Job
```

Comment:
I may not understand the purpose of this Job.  The scheduler script seems keep running.  I wonder if this should be Pod or Deployment.

Reply:
Yeah, maybe you are right. Probably it should be deployment

Reply:
converted to deployment